demo/src/app/components/accordion/accordion.component.ts

```diff
@@ -3,17 +3,20 @@ import { Component } from '@angular/core';
 import { ComponentDocsComponent, ExampleBoxComponent } from '../shared';
 import { AccordionBasicComponent, basicHtmlContent, basicTsContent } from './demos';
 
-const accordionDocs = require('./demos/ngb-accordion.json');
-const panelDocs = require('./demos/ngb-panel.json');
+const docs = require('../../../docs.json');
 
 @Component({
   selector: 'ngbd-accordion',
   template: require('./accordion.component.html'),
   directives: [AccordionBasicComponent, ComponentDocsComponent, ExampleBoxComponent]
 })
 export class AccordionComponent {
-  accordionDocs = accordionDocs;
-  panelDocs = panelDocs;
+  accordionDocs = docs['NgbAccordion'];
```

Comment:
it should - i don't think any of our keys are non-standard.  but i admittedly have no preference here.